secure_message/resources/threads.py

```diff
@@ -20,13 +23,68 @@ def get(thread_id):
         logger.info("Getting messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
 
         conversation = Retriever().retrieve_thread(thread_id, g.user)
+        conversation_metadata = Retriever().retrieve_thread_metadata(thread_id)
 
         logger.info("Successfully retrieved messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
         messages = []
         for message in conversation.all():
             msg = message.serialize(g.user, body_summary=False)
             messages.append(msg)
-        return jsonify({"messages": add_users_and_business_details(messages)})
+
+        if conversation_metadata.is_closed:
+            return jsonify({"messages": add_users_and_business_details(messages),
+                            "metadata": ThreadById._build_metadata(conversation_metadata)})
```

Comment:
Is there a reason for nesting this under a metadata property? An alternative could be:
```
{
    "is_closed": true,
    "closed_by": "someone",
    ...
    "messages": [
		{},
		{}
	]
}
```

Reply:
Oddly enough that was my first design, should've trusted my gut on that ^^